vmdk_plugin/main.go

```diff
@@ -97,8 +98,8 @@ func main() {
 	var driver volume.Driver
 
 	// Define command line options
-	driverName := flag.String("driver", "vmdk", "Volume driver")
-	logLevel := flag.String("log_level", "debug", "Logging Level")
+	driverName := flag.String("driver", vmdkDriver, "Volume driver")
```

Comment:
This change expects user to either configure or pass command line argument --driver=vsphere.

Any reason why we can't register both "vmdk" and "vsphere" pointing to same handlers? Ideally, with NO changes in installation process, following commands should work;

`docker volume create -d=vmdk v1`
Not sure if we could postfix warning that vmdk is deprecated and use vsphere instead.
`docker volume create -d=vsphere v1`

This can potentially open volume naming issues which I believe Docker tracks per Volume Driver. However, we don't expect user to mix & match vmdk/vsphere but rather to stick to one.  So, we should be fine.